cmd/zed/lake/branch/command.go

```diff
@@ -38,13 +39,13 @@ func init() {
 
 type Command struct {
 	lake      zedlake.Command
-	at        string
+	drop      bool
```

Comment:
Maybe call this `delete` instead since the all other text refers to the operation as "delete" not "drop".